src/quo/components/tags/context_tag/schema.cljs

```diff
@@ -0,0 +1,103 @@
+(ns quo.components.tags.context-tag.schema
+  (:require [malli.core :as malli]))
+
+(def ^:private ?base
+  [:map
+   [:type {:optional true}
+    [:maybe
+     [:enum :default :multiuser :group :channel :community :token :network :multinetwork :account
+      :collectible :address :icon :audio]]]
+   [:customization-color {:optional true} [:maybe :schema.common/customization-color]]
+   [:theme :schema.common/theme]
+   [:blur? {:optional true} [:maybe :boolean]]
+   [:state {:optional true} [:maybe [:enum :selected :default]]]])
+
+(def ^:private ?size
+  [:map
+   [:size {:optional true} [:maybe [:enum 24 32]]]])
+
+(def ^:private ?default
+  [:map
+   [:profile-picture {:optional true} [:maybe [:or :schema.common/image-source :string]]]
+   [:full-name {:optional true} [:maybe :string]]])
+
+(def ^:private ?multiuser
+  [:map
+   [:users {:optional true}
+    [:maybe
+     [:sequential
+      [:map [:profile-picture {:optional true} [:maybe [:or :schema.common/image-source :string]]]
```

Comment:
What if we changed the `schema.common/image-source` to accept strings ? 

This image source is used at multiple places.

Reply:
The current implementation aligns with the [React Native image source object](https://reactnative.dev/docs/0.72/images#source-as-an-object). Also, I believe in some places we are not accepting strings.

Reply:
If most places accept strings and the few exceptions can accommodate them as well, then updating the `image-source` implementation seems reasonable.

Reply:
@shivekkhurana @ajayesivan At the moment, I removed accepting the :string for the component and it seems to be fine. Let me know what do you think we should do